Automattic/jetpack
# Disallow posts with _term_meta post_type from syncing.

Add _term_meta to the disallowed post_types for syncing.

Fixes # https://wp.me/p3QzjZ-1cB

#### Changes proposed in this Pull Request:
Adds _term_meta to the disallowed post_types in Jetpack Sync.

#### Does this pull request change what data or activity we track or use?
Yes, it restricts the data we sync to WP.com.

#### Testing instructions:






* Apply patch to your Jetpack connected site
* Register the _term_meta post_type
* Create a new _term_meta post
* Verify the post is not synced to the WP.com Cache DB.
